notebooks/500_use_cases/501_dobot/501b_inference_with_a_robotic_arm.ipynb

```diff
@@ -1,14 +1,16 @@
 {
```

Comment:
<p>USB camera instead of US camera. Also I would like to include this: </p><p> </p><p>At this point we will work with the Robot and we need to keep the driver ins the same 501 folder, please copy and paste the files from /501_dobot/dobot_api to /501_dobot. If you don;t have that folder create make sure you are running the notebooks [501a]('link_of the notebook')</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/anomalib/pull/1084/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>